synced_versions_formulae.json

```diff
@@ -31,6 +32,8 @@
   ["logcli", "loki", "promtail"],
   ["mame", "rom-tools"],
   ["mecab-unidic", "mecab-unidic-extended"],
+  ["minizip", "zlib"],
```

Comment:
Currently, zlib is built using Wheezy but minizip is not so this one can't happen at the moment.

Reply:
Should probably document this somewhere so it doesn't happen again. Or perhaps audit for it.

Reply:
I imagine the easiest solution is to simply add a comment above the URL in `minizip`, like the ones that were in the `boost-*` formulae here.

Is there any visible criterion that can be checked/monitored to determine if/when the version can be synced for `minizip`/`zlib`? That is to say, if the comment uses a `# Please add to synced_versions_formulae.json once X` format, what would X be?